aviary/modules/binning/envs/comebin.yaml

```diff
@@ -0,0 +1,9 @@
+channels:
+  - pytorch
+  - nvidia
+  - conda-forge
+  - bioconda
+dependencies:
+  - comebin=1.0.*

```

Comment:
I'm imagining this will end up breaking at some point for someone without pinning an explicit version. But it's an optional binner so it's fine